Resources/ArmTemplates/test-parameters.json

```diff
@@ -0,0 +1,125 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "authentication.PushServiceClientId": {
+            "value": "00000000-0000-0000-0000-000000000000"
+        },
+        "authentication.PushServiceClientSecret": {
+            "value": "not-a-real-secret"
+        },
+        "authentication.Tenant": {
+            "value": "00000000-0000-0000-0000-000000000000"
+        },
+        "careersHelpline.AppIdUri": {
+            "value": "https://notatenant.mail.onmicrosoft.com/00000000-0000-0000-0000-000000000000"
+        },
+        "careersHelpline.Url": {
+            "value": "https://not-a-real-endpoint.azurewebsites.net/api/URLEndpoint"
+        },
+        "cosmosDbCollectionId": {
+            "value": "customers"
+        },
+        "cosmosDbDatabaseId": {
+            "value": "customers"
+        },
+        "cosmosDbEndpoint": {
+            "value": "https://dss-foo-shared-cdb.documents.azure.com:443"
+        },
+        "cosmosDbKey": {
+            "value": "not-a-real-secret"
+        },
+        "environmentName": {
+            "value": "foo"
+        },
+        "functionAppNames": {
+            "value": [
+                "dss-at-cpush-fa"
+            ]
+        },
+        "serviceBusConnectionString": {
+            "value": "sb://dss-foo-shared-ns.servicebus.windows.net/"
+        },
+        "sharedAppServicePlanName": {
+            "value": "dss-at-shared-asp"
+        },
+        "sharedAppServicePlanResourceGroup": {
+            "value": "dss-at-shared-rg"
+        },
+        "templateBaseUrl": {
+            "value": "https://raw.githubusercontent.com/SkillsFundingAgency/dss-contentpushservice/SDD-229-AddYamlPipeline/Resources/ArmTemplates"
```

Comment:
this will need changing in a subsequent PR

Reply:
noted, will do